packages/protocol/contracts/common/OwnerUUPSUpgradable.sol

```diff
@@ -44,7 +44,7 @@ abstract contract OwnerUUPSUpgradable is UUPSUpgradeable, OwnableUpgradeable {
         if (_loadReentryLock() == _TRUE) revert REENTRANT_CALL();
         _storeReentryLock(_TRUE);
         _;
-        _storeReentryLock(_FALSE);
+        _storeReentryLock(0);
```

Comment:
using 0 is the same as `clearing the storage`.

Reply:
Agreed that this is less confusing, only a single false value now instead of both `0` and `_FALSE`.